### ISSUE TITLE
Cannot open old state: No such file or directory (os error 2)

### ISSUE THREAD

**Issue by Minda1975** [leftwm/leftwm#1042]
### Describe the bug

Good day. Sorry for my bad English. Today I installed leftwm using the instructions on your github page (optional development installation). Installation went well, everything works fine. I checked the systemd log using `journalctl -p err..alert` and saw this entry `shell leftwm-worker[5737]: Cannot open old state: No such file or directory (os error 2)`. This message keeps repeating when I log in to a new session. I would like to know what the reason is and if this message can be avoided. I don't want this message to be repeated all the time and to keep this information from filling up my systemd log. I am using Debian 12, systemd version 252.5-2.

### Expected behavior (optional)

Systemd should keep clear.

### Steps to reproduce (optional)

1. Login to the session.
2. 2. Logout and check systemd log with command `journalctl -p err..alert` 

### Relevant log output (optional)

```text
мар 07 09:17:45 shell leftwm-worker[5737]: Cannot open old state: No such file or directory (os error 2)
мар 07 09:46:15 shell leftwm-worker[8677]: Cannot open old state: No such file or directory (os error 2)
```

### OS / Distro

Linux shell 6.1.0-5-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.12-1 (2023-02-15) x86_64 GNU/Linux

### Additional System Information (optional)

```text
Using systemd and started leftwm using lightdm.
```

### LeftWM Check

```text
:: LeftWM version: 0.4.1
:: LeftWM git hash: ec0923b
:: Enabled features: default journald-log lefthk lefthk-core tracing-journald
:: Checking feature dependencies . . .
    -> journald-log OK
    -> lefthk OK
:: Loading configuration . . .
    -> Configuration loaded OK 
:: Checking keybinds . . .
    -> All keybinds OK
:: Checking environment . . .
    -> Environment OK 
:: Checking theme . . .
ERROR: Could not parse theme file: 1:16: Expected option
```

**Comment by mautamu** [comment]
Source: https://github.com/leftwm/leftwm/blob/ec0923b32a69f4e1ae57b52b1bb88bd19bc21d8c/leftwm/src/config.rs#L578

This is somewhat expected, I guess? My reading of this is that it's supposed to be a trace to inform you that leftwm is "starting cold" (i.e. without a previous state like you'd see from a refresh). Perhaps we want to change this message to read something like "Starting leftwm fresh as no state file from a previous session exists"?

**Comment by Minda1975** [comment]
> This is somewhat expected, I guess? My reading of this is that it's supposed to be a trace to inform you that leftwm is "starting cold" (i.e. without a previous state like you'd see from a refresh). 

Thanks for the explanation.

> Perhaps we want to change this message to read something like "Starting leftwm fresh as no state file from a previous session exists"?

Yes, that would be a logical and better solution. But I have a question. Is it necessary for such trivial leftwm actions to be logged in the systemd log? Maybe it would be better if only serious leftwm errors were logged in the systemd log? And trivial actions like open old state would be written to .xsession-errors?  Would that be very difficult to do?

**Comment by VuiMuich** [comment]
I see this might be confusing.
Without looking at the code details I would propose a different approach:
This error makes sense in case the reload fails due to a broken or lost state file, so in case of a "cold start" I would like to try to suppress this message. Iirc there is some function in the core or std lib for such run once occasions and I believe for something we already make use of it.

**Comment by Minda1975** [comment]
> I would propose a different approach:
> In case of a "cold start" we could to try to suppress this message. Iirc there is some function in the core or std lib for such run once occasions and I believe for something we already make use of it.

Exactly. This would be the best solution, since making entries to the systemd log for the smallest leftwm action does not make sense. I believe that such entries should only be made when there is an error.

P.S.

Right now I'm forced to use commands `sudo journalctl --rotate` and `sudo journalctl --vacuum-time=1s` all the time as many records accumulate. Without these entries, the systemd log is completely clean. Pure as a baby's tear.
